README.md

```diff
@@ -318,7 +318,7 @@ Caching, Importing Blocks, and Block Information
   journaldb keccak-hasher len-caching-lock macros memory-cache memzero
   migration-rocksdb ethcore-network ethcore-network-devp2p panic_hook
   patricia-trie-ethereum registrar rlp_compress rlp_derive parity-runtime stats
-  time-utils triehash-ethereum unexpected parity-version
+  triehash-ethereum unexpected parity-version
```

Comment:
Just noticed we left a reference to the C bindings on line 299 when we removed them a few weeks back. Do you mind removing lines 299-302 here?

Reply:
done